### PR TITLE
Improve: buffer configuration infrastructure

### PR DESCRIPTION
**Pull request by yxieca**
**- What I did**
Add infrastructure of buffer configuration template base on existing code.

**- How to verify it**
The code has been tested on Arista7260CX3 platform as T0.

### CODE REVIEW

File: files/build_templates/buffers_config.j2 [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+            {%- elif speed == '50000' %}
+                {%- if PORT_50G.append(port) %}{%- endif %}
+            {%- elif speed == '40000' %}
+                {%- if PORT_40G.append(port) %}{%- endif %}
+            {%- elif speed == '25000' %}
+                {%- if PORT_25G.append(port) %}{%- endif %}
+            {%- elif speed == '10000' %}
+                {%- if PORT_10G.append(port) %}{%- endif %}
+            {%- else %}
+                {%- if default_set.append(port) %}{%- endif %}
```
Comment (yxieca) [inline files/build_templates/buffers_config.j2]: look into the possibility of looking up speed information from port_config.ini

File: files/build_templates/buffers_config.j2 [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        "{{ port }}|3-4": {
+            "profile" : "[BUFFER_PROFILE|ingress_lossless_profile_50g]"
+        },
+{% endfor %}
+{% for port in PORT_100G %}
+        "{{ port }}|3-4": {
+            "profile" : "[BUFFER_PROFILE|ingress_lossless_profile_100g]"
+        },
+{% endfor %}
+{% if port_names_all|length > 0 %}
```
Comment (yxieca) [inline files/build_templates/buffers_config.j2]: remove or duplicate below

File: files/build_templates/buffers_config.j2 [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+{% for port in PORT_50G %}
+        "{{ port }}|3-4": {
+            "profile" : "[BUFFER_PROFILE|ingress_lossless_profile_50g]"
+        },
+{% endfor %}
+{% for port in PORT_100G %}
+        "{{ port }}|3-4": {
+            "profile" : "[BUFFER_PROFILE|ingress_lossless_profile_100g]"
+        },
+{% endfor %}
```
Comment (yxieca) [inline files/build_templates/buffers_config.j2]: Remove default configuration for pg 3-4

File: device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-D108C8/buffers_defaults_t0.j2 [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+            "mode": "dynamic"
+        },
+        "egress_lossless_pool": {
+            "size": "42349632",
+            "type": "egress",
+            "mode": "static"
+        }
+    },
+    "BUFFER_PROFILE": {
+        "ingress_lossless_profile_50g": {
```
Comment (vharish02) [inline device/arista/x86_64-arista_7260cx3_64/Arista-7260CX3-D108C8/buffers_defaults_t0.j2]: Hi Ying, BUFFER_PROFILE section should have default values and not specific to any speed isn't?

**Pull request merged**